tests/device_handler.py

```diff
@@ -49,12 +53,35 @@ def __init__(self, client: "Client", nowait: bool = False) -> None:
     def _configure_client(self, client: "Client") -> None:
         self.client = client
         self.client.ui = NullUI  # type: ignore [NullUI is OK UI]
+        self.client.button_callback = self.client.ui.button_request
         self.client.watch_layout(True)
-        self.client.debug.input_wait_type = DebugWaitType.CURRENT_LAYOUT
```

Comment:
Not sure why it was removed...

[Some of the tests](https://github.com/trezor/trezor-firmware/actions/runs/16140105209/job/45581520204) rely on having non-`IMMEDIATE` wait type, in order to synchronize the client-side test flow with device-side layouts' changes - so I would suggest keeping it.